packages/server/src/services/Service.js

```diff
@@ -1,13 +1,15 @@
-import { camelize } from '@ditojs/utils'
+import { camelize, hyphenate } from '@ditojs/utils'
 
 export class Service {
   initialized = false
+  #loggerName = null
```

Comment:
In JS you don't need to declare it here if you set it in the constructor, you're essentially setting it twice in each initialisation.

Reply:
Right, I need to declare it but not to initialise it

Reply:
Right, the declaration bit is a bit unusual for JS, but apparently needed for private properties.